### PR TITLE
Allow Users to login from header

### PR DESCRIPTION
**Pull request by dhaneshnm**
- Add login flow to the login button in the header.

Fixes BlueBrain/nexus/issues/1674

### CODE REVIEW

**Comment by dhaneshnm** [comment]
Screenshot 2020-10-15 at 07 10 21

**Comment by dhaneshnm** [comment]
Screenshot 2020-10-15 at 07 35 52

**Review by Stafeeva — DISMISSED** [review]

File: src/shared/layouts/FusionMainLayout.tsx [hunk 1]
```diff
@@ -22,15 +22,64 @@ const { Sider, Content } = Layout;
 
 const logo = require('../images/logoDarkBg.svg');
 
+const loginCallBack = async (userManager: UserManager) => {
+  try {
+    // setPreferredRealm(preferredRealm);
```
Comment (kenjinp) [inline src/shared/layouts/FusionMainLayout.tsx:27]: some old comment?

**Review by kenjinp — CHANGES_REQUESTED** [review]

**Review by kenjinp — APPROVED** [review]

**Review by Stafeeva — APPROVED** [review]
Tested in Chrome 💯

**Review by kenjinp — APPROVED** [review]

**Pull request merged**